packages/js/components/src/tag/index.tsx

```diff
@@ -92,6 +105,12 @@ const Tag = ( {
 };
 
 Tag.propTypes = {
```

Comment:
I don't think you need propTypes anymore and the inline comments can be moved to the type def so that it shows up in the mouseover for the type 😀